### DISCUSSION TITLE
does this driver supports ubuntu 23.04?

### DISCUSSION THREAD

**Discussion by mohammad-zadeed** [Mange/rtl8192eu-linux-driver#318] [Q&A] [answered]
1)does this driver supports ubuntu 23.04?
2)can i use it on tp link wn823n usb wifi dongle?
3)how to use htis drivers in fedora 37?

**Comment by SatanicRaw** [comment, marked as answer]
1) Yes, the driver is compiled for the kernel up to 6.3
2) Not to say right away, you need to look at the VID / PID using the lsusb command.
3) Yes, standard kernel (5.19) is supported

**Comment by travahacker** [comment]
I had this issue with ubuntu 23.10

https://github.com/Mange/rtl8192eu-linux-driver/issues/336
